### PR TITLE
Add favicon to codesearch. Fixes #1388.

### LINKED ISSUE #1388: CodeSearch: Favicon missing
screen shot 2015-01-01 at 2 18 58 pm

You can fix this by adding a `sourceIconUrl` such as https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/currency/currency.js#L128

### PR DESCRIPTION
**Pull request by MoriTanosuke**
I added the `sourceIconUrl` as recommended in #1388. Also changed the `sourceUrl` to use https and the same URL.

### CODE REVIEW

File: share/spice/code_search/code_search.js [hunk 1]
```diff
@@ -23,7 +23,8 @@
                 record_keys: keys
             },
             meta: {
-                sourceUrl: 'http://searchco.de/?q=' + query + '&cs=true',
+                sourceUrl: 'https://searchcode.com/?q=' + query + '&cs=true',
+                sourceIconUrl: 'https://searchcode.com/static/favicon.ico'
```
Comment (jagtalon) [inline share/spice/code_search/code_search.js]: Missing a comma

**Comment by jagtalon** [comment]
Looks like it's still not appearing

screen shot 2015-01-05 at 12 05 44 pm

**Comment by MoriTanosuke** [comment]
Ups, thanks for the comma. Fixed that in my fork too.

For me it's working, just tried it in my codio box: 

screenshot - 05 01 2015 - 19 54 02

**Comment by jagtalon** [comment]
@MoriTanosuke dang I see what the problem now--it looks like it's working, but it fails on Chrome :+1:

**Pull request merged**